### PR TITLE
WIP: AGMS-315 - diagram of sync and sync content update

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
https://issues.jboss.org/browse/AGMS-315

## What
https://issues.jboss.org/browse/AGMS-315

## Why
https://issues.jboss.org/browse/AGMS-315

## How
https://issues.jboss.org/browse/AGMS-315

### CODE REVIEW

**Comment by jstaffor** [comment]
@finp - would you mind taking a look into the upstream/downstream tag functionality on this pr? I've tried a number of different permutations that were based on existing snippets of content. however, I cannot seem to display the upstream image and not the downstream image.

File: modules/ROOT/pages/data-sync.adoc [hunk 1]
```diff
@@ -15,22 +15,9 @@ include::_partials/attributes.adoc[]
 
 [[introduction]]
 
-Data Sync is a JavaScript framework that enables a developer add real time synchronization and enhanced offline capabilities to mobile applications.
+include::_partials/sync/introducing-data-sync.adoc[leveloffset=2,tags=!excludeDownstream]]
```
Comment (finp) [inline modules/ROOT/pages/data-sync.adoc]: @jstaffor  you've got an extra close bracket there

]]

Comment (finp) [reply modules/ROOT/pages/data-sync.adoc]: same problem below, search for `]]`, just a copy paste error

File: modules/ROOT/pages/_partials/sync/data-sync-case-study-generic.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[id="data-sync-case-study-generic-{context}"]
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/data-sync-case-study-generic.adoc]: spoke to @wei-lee  need to remove this case study

File: modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+// end::excludeDownstream[]
+
+// tag::excludeUpstream[]
+
+image::data-sync-technical-overview.png[Data Sync Technical Overview]
+
+// end::excludeUpstream[]
+
+.Data Sync case study
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc]: remove this case study too

File: modules/ROOT/pages/_partials/sync/introducing-data-sync.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Data Sync is a JavaScript framework that enables a developer to add real time data synchronization to both mobile and web clients.
+The Data Sync framework also provides offline capabilities that allow a client to continue operating offline and once connectivity is re-established, the client is automatically synchronized.
+An app built using the Data Sync framework typically connects to a data source for data persistence however, an app built using the Data Sync framework works without a data source.
+
+An app built using the Data Sync framework comprises of two components:
+
+* The Sync Client is a JavaScript client offering client side extensions and server side integration. The Sync Client can be integrated into frameworks such as React and Angular.
+
+// tag::excludeDownstream[]
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/introducing-data-sync.adoc]: remove diagrams

File: modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
+[id="data-sync-technical-overview-{context}"]
+= Data Sync Technical Overview
+
+This section describes the technical aspects of Data Sync.
+
+// tag::excludeDownstream[]
+
+image::data-sync-technical-overview.png[Data Sync Technical Overview]
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc]: @jstaffor  do we have a source for this?

Comment (jstaffor) [reply modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc]: You can find the image here @finp : https://docs.google.com/presentation/d/1CUd1Gcx3mGKVjKAURVJ49so0NANG0snqwIJU-YPYKAM/edit#slide=id.g50c800615d_7_169

File: modules/ROOT/pages/_partials/sync/data-sync-case-study-generic.adoc (hunk 2 above)
Comment (jstaffor) [reply modules/ROOT/pages/_partials/sync/data-sync-case-study-generic.adoc]: Complete in this commit: https://github.com/aerogear/mobile-docs/pull/691/commits/48a7e8cda0cbedc0b2f7e5bc5e91730af1cb6899

File: modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc (hunk 3 above)
Comment (jstaffor) [reply modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc]: Completed as part of https://github.com/aerogear/mobile-docs/commit/48a7e8cda0cbedc0b2f7e5bc5e91730af1cb6899

File: modules/ROOT/pages/_partials/sync/introducing-data-sync.adoc (hunk 4 above)
Comment (jstaffor) [reply modules/ROOT/pages/_partials/sync/introducing-data-sync.adoc]: Completed as part of https://github.com/aerogear/mobile-docs/pull/691/commits/6a273828dc7f778021af3fb6ff9f967f9003c58e

File: modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc (hunk 5 above)
Comment (jstaffor) [reply modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc]: We do. The main source for all the image work pertaining to this PR can be seen here: https://issues.jboss.org/browse/AGMS-315

It was provided to us by Engineering. If you need anything else, let me know :+1:

**Comment by jstaffor** [comment]
Returning to WIP due to work on the images. It's not yet clear on how to progress.

**Comment by jstaffor** [comment]
Supporting material sent to CCS Designers this morning.

**Comment by finp** [comment]
closing this PR, as I used this as the basis for data-sync-reorg, see https://github.com/finp/mobile-docs/commits/data-sync-reorg

**Pull request closed without merging**